src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -342,10 +359,10 @@ def _update_employer_metadata(
         employer_ids = pop.loc[rows_to_update, "employer_id"]
 
         pop.loc[rows_to_update, "employer_address_id"] = employer_ids.map(
-            self.businesses.set_index("employer_id")["employer_address_id"].to_dict()
+            self.businesses["employer_address_id"].to_dict()
```

Comment:
I know you didn't introduce this, but I'm pretty sure `.to_dict()` is not necessary here.